tripal_chado/src/api/ChadoSchema.php

```diff
@@ -218,7 +218,7 @@ public function getTableNames($include_custom = FALSE) {
     // now add in the custom tables too if requested
     // @todo change this to the variable once custom tables are supported.
     if (FALSE) {
-      $sql = "SELECT table FROM {tripal_custom_tables}";
+      $sql = "SELECT table FROM [tripal_custom_tables]";
       $resource = $this->connection->query($sql);
```

Comment:
Don't we need to make sure we actually get here? And I think it's best to use 0 here to indicate Drupal rather then the older [ ] approach :thinking:

That said, this is the old ChadoSchema class that will be deprecated. Not the new one that is part of Tripal DBX.

```suggestion
    // Now add in the custom tables too if requested.
    if ($include_custom) {
      $sql = "SELECT table FROM {0:tripal_custom_tables}";
      $resource = $this->connection->query($sql);
```

Reply:
Can't commit suggestion "Applying suggestions on deleted lines is currently not supported." but I will add this...

Reply:
I have reverted this to never run, because tests are failing, but it's not worth the time to figure out why for a deprecated function that never executed anyway